datafusion/physical-expr-common/src/sort_expr.rs

```diff
@@ -264,9 +267,55 @@ pub type LexOrdering = Vec<PhysicalSortExpr>;
 /// a reference to a lexicographical ordering.
 pub type LexOrderingRef<'a> = &'a [PhysicalSortExpr];
```

Comment:
Perhaps we could follow a similar pattern for LexOrderingRef `too`

Reply:
BTW having a real struct will make printing these structures much easier -- e.g. I had to do some akward workarounds to implement `Display` or `LexOrdering` in https://github.com/apache/datafusion/pull/12590